components/tabs/stories/tabs-horizontal.stories.js

```diff
@@ -30,11 +28,110 @@ export default {
   }
 };
 
-export const Default = Template.bind({});
-Default.args = horizontalProps;
+const chromaticKitchenSink = ({
+	...args
+}) => {
+	return html`
+		<div style="display: flex; flex-direction: column; gap: 2rem;">
+    <h2>Horizontal Default</h2>
+			${Template({
+        ...args
+			})}
+			${Template({
+        ...args,
+        labelWithIcons: true
+       })}
+      ${Template({
+        ...args,
+        iconOnly: true
+        })}
```

Comment:
Since these are repeated in the 3 contexts below, can we pull that out into its own function? Maybe those are named `TabsGroup`? Also, since this same set appears in the vertical set, maybe move that definition to index.js and export it for use in both files?